src/_companies/north-america/canada/british-columbia/netsign/index.md

```diff
@@ -0,0 +1,24 @@
+---
+name: "Netsign"
+
+continent: "North America"
+country: "Canada"
+region: "British Columbia"
+city: "Vancouver"
+
+logo: logo.png
+
+website: https://netsign.com
+
+work_arrangements:
+    - "remote"
+
+stack:
+  - "rails"
+  - "postgres"
+  - "stimulus"
+
+created_at: 2023-03-28 00:26:23
+---
+Netsign, founded in 1999, is a dynamic group of senior freelancers who provide product development consulting. We’ve worked on dozens of rails apps collectively for companies like Unbounce, ZenDesk, and Climate Smart. Do you need a team of senior developers that can quickly take your idea from a napkin sketch to fully realized web application that makes money for your business? We can provide strategy, product management, design, development, mentoring and maintenance for your project. Lean on us for all of it or just part of the puzzle.
```

Comment:
Could you change this to be in the third person? That way it'll be consistent with all other company descriptions :)

Reply:
Sure. Done!